rulesets/NI.Tests.ruleset

```diff
@@ -0,0 +1,156 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="NI.Tests.ruleset" Description="Code analysis rule overrides for NI Test projects." ToolsVersion="14.0">
+  <IncludeAll Action="Warning" />
+  <Include Path=".\NI.ruleset" Action="Default" />
+  <Rules AnalyzerId="Microsoft.Analyzers.ManagedCodeAnalysis" RuleNamespace="Microsoft.Rules.Managed">
+    <Rule Id="CA1001" Action="None" />
+    <Rule Id="CA1002" Action="None" />
+    <Rule Id="CA1005" Action="None" />
+    <Rule Id="CA1008" Action="None" />
+    <Rule Id="CA1017" Action="None" />
+    <Rule Id="CA1018" Action="None" />
+    <Rule Id="CA1019" Action="Warning" />
+    <Rule Id="CA1024" Action="None" />
+    <Rule Id="CA1026" Action="None" />
+    <Rule Id="CA1027" Action="None" />
+    <Rule Id="CA1030" Action="None" />
+    <Rule Id="CA1032" Action="None" />
+    <Rule Id="CA1034" Action="None" />
+    <Rule Id="CA1040" Action="None" />
+    <Rule Id="CA1051" Action="None" />
+    <Rule Id="CA1054" Action="None" />
+    <Rule Id="CA1064" Action="None" />
+    <Rule Id="CA1065" Action="None" />
+    <Rule Id="CA1304" Action="None" />
+    <Rule Id="CA1305" Action="None" />
+    <Rule Id="CA1308" Action="None" />
+    <Rule Id="CA1405" Action="None" />
+    <Rule Id="CA1702" Action="None" />
+    <Rule Id="CA1704" Action="None" />
+    <Rule Id="CA1707" Action="None" />
+    <Rule Id="CA1709" Action="None" />
+    <Rule Id="CA1710" Action="None" />
+    <Rule Id="CA1711" Action="None" />
+    <Rule Id="CA1712" Action="None" /> <!-- RoslynTransition: Should be "Warning". Tracked by Dev Efficiency: https://trello.com/b/gtoe4Of2/iteration-board -->
```

Comment:
Should we track this a different way before posting this on GitHub? Who should be tracking this?

Reply:
I am deleting all of these. Almost all of them are stale and we should just track everything as we find it from here on out in github or at least in the coding practices chapter backlog.